TFT/src/User/Menu/common.c

```diff
@@ -82,6 +82,39 @@ const ITEM itemExtLenSteps[ITEM_EXT_LEN_NUM] =
 // List for extruder length/distance change steps
 const float extlenSteps[ITEM_EXT_LEN_NUM] = {1.0f, 5.0f, 10.0f, 100.0f, 200.0f};
 
+const uint16_t iconToggle[ITEM_TOGGLE_NUM] =
+{
+  ICONCHAR_TOGGLE_OFF,
+  ICONCHAR_TOGGLE_ON
+};
+
+const LABEL itemToggle[ITEM_TOGGLE_NUM] =
+{
+  LABEL_OFF,
+  LABEL_ON
+};
+
+const LABEL itemToggleAuto[ITEM_TOGGLE_AUTO_NUM] =
+{
+  LABEL_OFF,
+  LABEL_ON,
+  LABEL_AUTO
+};
+
+const LABEL itemToggleSmart[ITEM_TOGGLE_SMART_NUM] =
+{
+  LABEL_OFF,
+  LABEL_ON,
+  LABEL_SMART
+};
+
+const LABEL itemMoveSpeed[ITEM_SPEED_NUM] =
+{
+  LABEL_SLOW_SPEED,
+  LABEL_NORMAL_SPEED,
+  LABEL_FAST_SPEED
+};
+
 // Check time elapsed against the time specified in milliseconds for displaying/updating info on screen
```

Comment:
please do not add items in common.c/common.h files if they are used in only one file.

Reply:
hmm I re-used itemToggle in Z offset menu. IMHO those values could be used in many other future menus. However, if you don't agree, I can move them in FeatureSettings, Let me know

Reply:
when the future comes they can be placed here. For now, only one file is using them

Reply:
moved on FeatureSettings